app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -59,9 +58,11 @@ def switch!
         datacite_endpoint.switch!
         Settings.switch!(name: locale_name, settings: settings)
         switch_host!(cname)
+        setup_tenant_cache(cache_api?)
       end
 
       def reset!
+        setup_tenant_cache(cache_api?)
```

Comment:
This should be `false` no?

Reply:
No I think I just want to make everything dependent on the Flipflop otherwise you turn off the Cashing after every request